demo_scripts/simdem/syntax/README.md

```diff
@@ -28,21 +28,24 @@ aware of. They are detailed in the next few sections.
 ## Code Blocks
 
 In SimDem a code block is marked in exactly the same way it is in
-Markdown, that is with three backticks (``````). Unless a Code Block
+Markdown, that is with three backticks (``````), followed by a language hint. 
+For the code block to be considered executable it is good practice to use
+the `bash` hint. However, the absence of a `bash` hint will result in the
+assumption that it is `bash` code. Unless a Code Block
 is marked as a Results block (see next section) it is assumed that
 this is executable code. SimDem will execute each line individually.
 
 For example:
 
-```
+```bash
 # This is a code block, this comment will be ignored by SimDem
 echo "This command will be 'typed' and executed."
 ```
 
 Code blocks can optionally be indented to make the markdown more 
 readable. For example:
 
-    ```
+    ```bash
     echo "Hello world indent"
     ```
```

Comment:
This does still does not have the expected behavior. When I pulled the PR and did testing the program ended immediately after 'hello world indent' see screenshot 
![image](https://user-images.githubusercontent.com/91570951/180308312-e4f13f41-44b4-4a33-9c8a-b46872665467.png)

Reply:
That is weird. It 100% worked. I did squash some commits. I wonder if I inadvertently included some in progress work on the expected_similarity change, because I do have thus break in my working branch on that.

Anyway, good catch.